src/Swarm/TUI/Controller.hs

```diff
@@ -241,7 +241,7 @@ runFrame = do
   runFrameTicks (fromNanoSecs dt)
 
 ticksPerFrameCap :: Int
-ticksPerFrameCap = 30
+ticksPerFrameCap = 32
```

Comment:
There's no reason to change this.  As I mention below, this number has nothing to do with the ticks per second.